docs/fsharp-cheatsheet.md

```diff
@@ -368,32 +488,39 @@ Single-case discriminated unions are often used to create type-safe abstractions
     let orderId = Order "12"
 
     // Use pattern matching to deconstruct single-case DU
-    let (Order id) = orderId
+    let (Order id) = orderId  // id = "12"
 
-<div id="statically-resolved-type-parameters"></div>
+See [Discriminated Unions](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/discriminated-unions) to learn more.
 
-## Statically Resolved Type Parameters
+<div id="pattern-matching"></div>
 
-A *statically resolved type parameter* is a type parameter that is replaced with an actual type at compile time instead of at run time. They are primarily useful in conjunction with member constraints.
+## Pattern Matching
```

Comment:
should this move earlier?
perhaps this would allow to convey that deconstruction in `let` and arg lists is kinda also pattern matching

Reply:
I split up the pattern matching and this ended up being the final catch. The intent was, "we covered X, Y, Z in those sections; here's value, OR, and AND."

Reply:
Ah, that makes sense - perhaps slip a 'misc' into the title and/or add an intro that mentions "yes, there is `match`, but you'll find this in `let`, `function` and other places ... its pervasive.